content/src/roadmaps/tasks/site-inspection.md

```diff
@@ -14,6 +14,8 @@ callToActionLink: ""
 callToActionText: ""
 ---
 
-> **Once you complete this task, you will have:**
->
-> - A cosmetology shop license
+:::callout{ header="Once you complete this task, you will have:" icon="false" calloutType="conditional" }
```

Comment:
[dust] Not part of the ticket, but this is _very_ similar to the default text. Might want to check with Content to see if they just want the default.